runelite-api/src/main/java/net/runelite/api/ItemContainer.java

```diff
@@ -31,4 +31,6 @@
 	 * @return
 	 */
 	Item[] getItems();
+
+	int[] getItemIds();
```

Comment:
Why is getItems() not enough?

Reply:
Because it does a Arrays.equals check and Item doesn't implement equals, although I guess equals could be implemented in a mixin